src/components/InRequest/InRequest.tsx

```diff
@@ -1,654 +1,57 @@
-                  control={control}
-                  rules={{
-                    required: "Previous Organization is required",
-                  }}
-                  render={({ field }) => (
-                    <Input
-                      {...field}
-                      aria-describedby="prevOrgErr"
-                      id="prevOrgId"
-                    />
-                  )}
-                />
-                {errors.prevOrg && (
-                  <Text id="prevOrgErr" className={classes.errorText}>
-                    {errors.prevOrg.message}
-                  </Text>
-                )}
-              </>
-            )}
-          </>
-        )}
-        {(empType === EMPTYPES.Civilian || empType === EMPTYPES.Military) && (
-          <>
-            <Label htmlFor="newToBaseAndCenterId">
-              Is Employee new to WPAFB and AFLCMC?
-            </Label>
-            <Controller
-              name="isNewToBaseAndCenter"
-              control={control}
-              rules={{
-                required: "Selection is required",
-              }}
-              render={({ field }) => (
-                <RadioGroup
-                  {...field}
-                  aria-describedby="isNewToBaseAndCenterErr"
-                  id="newToBaseAndCenterId"
-                >
-                  <Radio key={"yes"} value={"yes"} label="Yes" />
-                  <Radio key={"no"} value={"no"} label="No" />
-                </RadioGroup>
-              )}
-            />
-            {errors.isNewToBaseAndCenter && (
-              <Text id="isNewToBaseAndCenterErr" className={classes.errorText}>
-                {errors.isNewToBaseAndCenter.message}
-              </Text>
-            )}
-          </>
-        )}
-
-        {empType === EMPTYPES.Contractor && (
+  if (!request.isLoading && request.data) {
+    return (
+      <>
+        <InRequestViewCompact formData={request.data} />{" "}
+        {isSupervisor && (
           <>
-            <Label htmlFor="hasExistingCACId">
-              Does the Support Contractor have an Existing CAC?
-            </Label>
-            <Controller
-              name="hasExistingCAC"
-              control={control}
-              rules={{
-                required: "Selection is required",
-              }}
-              render={({ field }) => (
-                <RadioGroup
-                  {...field}
-                  aria-describedby="hasExistingCACErr"
-                  id="hasExistingCACId"
-                >
-                  <Radio key={"yes"} value={"yes"} label="Yes" />
-                  <Radio key={"no"} value={"no"} label="No" />
-                </RadioGroup>
-              )}
+            <Button
+              appearance="primary"
+              className="floatRight"
+              onClick={showEditPanel}
+            >
+              Edit
+            </Button>
+            <InRequestEditPanel
+              onEditSave={hideEditPanel}
```

Comment:
The function provided here will need to do more than just hide the Edit Panel -- It needs to inform the compact view that the underlying data has changed.  Could be addressed via react-query update function to tell it to refetch -- or ideally do a mutation that just puts the new data in there.